careers/software-engineer.md

```diff
@@ -4,52 +4,56 @@ The goal of the Software Engineer path is to train engineers to master the funda
+**Communication:** They are able to effectively review code in their area of expertise and regularly point out opportunities for improvement in the code that they review. They proactively point out concerns with proposed or current sprint work as soon as they see them. They can articulate the tradeoffs of different approaches to solving a problem.
 
-By the time an engineer is ready to be promoted to Engineer III they will have focused on a technology or functional area as their expertise and become capable of mentoring interns and new engineers in these areas.
+**Collaboration:** They are mindful of the work that others are doing and can critique or suggest improvements to the proposed plans of others. They get help from others when stuck, but are also capable of helping unstick their peers. They are capable of effectively running small to medium-sized projects with regular coaching.
 
 ## Engineer III
 
-**Technical Ability:** An Engineer III can consistently be relied on to take on an undefined project through to execution while fully and effectively utilizing the skills and intelligence of the team. An Engineer III has the ability to provide an answer to most technical questions that the team may have, but they make a choice to exercise restraint and ask questions to help other team members reach their own conclusions.
+An engineer III is capable of taking on complex technical problems and coordinating work spanning several months.
 
-**Technical Scope:** An Engineer III takes ownership over the "how" of the feature they implement with a "buck stops here" attitude. If an issue/bug does arise, they see developing a solution to that problem as their personal responsibility. An Engineer III actively seeks out knowledge about the risks and challenges in our business, using that knowledge to guide decision-making
+**Technical Knowledge:** They have a robust understanding of their technical specialty, and a working knowledge of other domains. They can solve most problems in their domain. Their code is well-abstracted and performant as a rule and rarely needs a significant refactor at the PR stage. If an unexpected issue/bug does arise, they see developing a solution to that problem as their personal responsibility.
 
-**Estimation:** An Engineer III Is able to provide reliable estimates for features that take longer than a single sprint to complete. Consistently delivers their work within a small margin of error to their original estimation, allowing the team to plan effectively and rely on the timely delivery of work.
+**Productivity:** They are capable of delivering a large volume of work regardless of the complexity/uncertainty of the work. They provide reliable estimates for features that take longer than a single sprint to complete.
```

Comment:
I don't feel like this section is totally accurate. AFAIK we don't assign different capacities to engineers at different levels, so the "volume of work" wouldn't really change between levels, right? And we generally don't have tickets that take longer than a sprint to complete, so does `They provide reliable estimates for features that take longer than a single sprint to complete` refer to being a DRI?

Reply:
> Does `They provide reliable estimates for features that take longer than a single sprint to complete` refer to being a DRI?

Yes. Or any other technical planning

Reply:
> AFAIK we don't assign different capacities to engineers at different levels, so the "volume of work" wouldn't really change between levels, right?

Not on a sprint-by-sprint basis, no. That said, I would expect EN3s to be much more reliable with their work than EN2 or EN1. Aggregated over the course of a year, this does lead to a higher overall volume of work. Put another way... we give everyone (roughly) the same number of points. However, an EN1 is way more likely to get stuck, flounder, make questionable decisions, have to redo work after code review, etc. By EN3, these hiccups in productivity should mostly be ironed out, even with highly complex work. They should be able to reliably crank through a lot of work without difficulty.